### ISSUE TITLE
Can't use landscape on an screen copy file

### ISSUE THREAD

**Issue by ozbillwang** [coinbase/terraform-landscape#41]
I run the `terraform plan` in a wrapper script, then I want to use `landscape` to reformation the output. 

I copy and paste the screen to a file, then `cat output.txt |landscape`, doesn't work. 
```
$ cat output.txt |landscape --trace
/usr/local/lib/ruby/gems/2.4.0/gems/terraform_landscape-0.1.9/lib/terraform_landscape/terraform_plan.rb:36:in `from_output': Expected one of [a-zA-Z0-9_-], '.' at line 1, column 12 (byte 12) after + terraform (TerraformLandscape::TerraformPlan::ParseError)
```

Can you add this feature that I can easily run landscape?

(I copy the exact same part of `terraform plan` output on screen)

**Comment by hposca** [comment]
Have you tried redirecting the output of the plan to a file?

You can either use `terraform plan > output.txt` or `terraform plan | tee output.txt` (if you want to follow the execution on screen).

Then you can _cat_ it with `cat output.txt | landscape`

**Comment by krobertson** [comment]
@ozbillwang from the sound of it, it seems like you're on Terraform 0.10.4 or newer, and from the output, you're on terraform-landscape 0.1.9. There was an issue where newer Terraform versions changed the output slightly and this was fixed in terraform-landscape 0.1.11.

I'f recommend going to the latest version (currently 0.1.15) to ensure you have the latest fixes for some of the output related issues.

Otherwise, I can confirm that on the latest versions of both, outputting to a file and using that through terraform-landscape works correctly.

```
$ terraform plan > output.txt
$ cat output.txt | landscape
```
